notebooks/timelapse.ipynb

```diff
@@ -423,9 +395,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "timelapse_task_simple = TimeLapseTask(timelapse_dir, 'timelapse_simple.gif', fps=6, brightness_factor=3)\n",
-    "timelapse_task_filtered = TimeLapseTask(timelapse_dir, 'timelapse_filtered.gif', fps=6, brightness_factor=3)\n",
-    "timelapse_task_filtered_coregistered = TimeLapseTask(timelapse_dir, 'timelapse_filtered_coregistered.gif', fps=6, brightness_factor=3)"
+    "timelapse_task_simple = TimeLapseTask(TIMELAPSE_DIR, 'timelapse_simple.gif', fps=6, brightness_factor=3)\n",
+    "timelapse_task_filtered = TimeLapseTask(TIMELAPSE_DIR, 'timelapse_filtered.gif', fps=6, brightness_factor=3)\n",
+    "timelapse_task_filtered_coregistered = TimeLapseTask(TIMELAPSE_DIR, 'timelapse_filtered_coregistered.gif', fps=6, brightness_factor=3)"
```

Comment:
These lines are a bit too long for my taste. Perhaps they could be split between multiple lines?